executor/runcexecutor/executor_linux.go

```diff
@@ -117,37 +105,26 @@ func (w *runcExecutor) callWithIO(ctx context.Context, id, bundle string, proces
 		})
 	}
 
+	started := make(chan int)
```

Comment:
nit: a buffered chan maybe as doesn't look like there is a need for write to block